fxsampler/src/main/java/fxsampler/FXSamplerProject.java

```diff
@@ -7,18 +33,25 @@
     /**
      * Returns the pretty name of the project, e.g. 'JFXtras' or 'ControlsFX'
      */
-    public String getProjectName();
-    
+    String getProjectName();
+
     /**
      * All samples should be beneath this base package. For example, in ControlsFX,
      * this may be 'org.controlsfx.samples'.
      */
-    public String getSampleBasePackage();
-    
+    String getSampleBasePackage();
+
     /**
      * Node that will be displayed in welcome tab, when project's root is
      * selected in the tree. If this method returns null, default page will 
      * be used
      */
-    public WelcomePage getWelcomePage();
+    WelcomePage getWelcomePage();
+
+    /**
+     * Module name of the project for which the sampler is to be used.
+     * For example, in case of ControlsFX this is "org.controlsfx.controls".
+     * Can be left blank if the sampler application is running on classpath.
+     */
+    String getModuleName();

```

Comment:
This will break existing projects that upgrade this dependency, won't it? 
If blank is an option, maybe you could provide a default empty implementation?

Reply:
Good idea!

Reply:
On a second thought, this can be converted to a default method and some Javadoc can be added to `Samples#getControlStylesheetURL` method explaining the importance of overloading it.

Reply:
Yes, that's what I meant with default empty implementation.